src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { Meta, Story } from "@storybook/react";
+
+import { Badge as BadgeComponent } from "./index";
+import { BaseBadgeProps } from "./type";
+
+export default {
+  title: "Components/Badge",
+  component: BadgeComponent,
+  argTypes: {
+    variant: {
+      options: ["light", "neutral", "info", "success", "warning", "critical"],
+      control: { type: "radio" },
+    },
+    inverted: {
+      options: [true, false],
+      control: { type: "boolean" },
+    },
+    suffix: {
+      options: [true, false],
+      control: { type: "boolean" },
+    },
```

Comment:
suffix boolean, referring to previous comments

Reply:
diatas gue balesnya ya

Reply:
okeh